### PR TITLE
Cabana: support multiple instances

### PR DESCRIPTION
**Pull request by deanlee**

### CODE REVIEW

File: tools/cabana/cabana.cc [hunk 1]
```diff
@@ -22,12 +24,22 @@ int main(int argc, char *argv[]) {
     cmd_parser.showHelp();
   }
 
+  QString uuid =  QUuid::createUuid().toString(QUuid::WithoutBraces);
```
Comment (adeebshihadeh) [inline tools/cabana/cabana.cc:27]: let's put this helper in `common/`

Comment (deanlee) [reply tools/cabana/cabana.cc:27]: do you mean cabana/common or somewhere else such as /ui/qt/util.cc?

Comment (adeebshihadeh) [reply tools/cabana/cabana.cc:27]: the root common/

**Comment by deanlee** [comment]
I'm not sure if it's appropriate to put it in qtutil.h like this

File: tools/cabana/cabana.cc (hunk 1 above)
Comment (deanlee) [reply tools/cabana/cabana.cc:25]: I have reverted https://github.com/commaai/openpilot/pull/26286/commits/19ef28945e8647724bbfc6c8469a664a58c09f16 , how about keep this pr as it is ,and move to common next time?

Comment (adeebshihadeh) [reply tools/cabana/cabana.cc:25]: sure, but I meant make a common function to setup a prefix environment

**Pull request merged**